MiniApp/Classes/RealMiniApp.swift

```diff
@@ -31,7 +31,7 @@ internal class RealMiniApp {
         return miniAppInfoFetcher.getInfo(miniAppId: miniAppId, apiClient: self.miniAppClient, completionHandler: completionHandler)
     }
 
-    func createMiniApp(appInfo: MiniAppInfo, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void) {
+    func createMiniApp(appInfo: MiniAppInfo, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void, messageInterface: MiniAppMessageProtocol) {
```

Comment:
I think that the `messageInterface` needs to be passed to `displayer.getMiniAppView` so that it can be used from the `WKScriptMessageHandler`, right?

Reply:
Done